apex/contrib/examples/nccl_allocator/cache.py

```diff
@@ -0,0 +1,54 @@
+import os
+import torch
+import apex.contrib.nccl_allocator as nccl_allocator
+from pynvml.smi import nvidia_smi
```

Comment:
is this https://pypi.org/project/nvidia-ml-py/?

Reply:
I believe it is this one: https://pypi.org/project/pynvml/